language-support/hs/bindings/src/DA/Ledger/Services/LedgerConfigurationService.hs

```diff
@@ -0,0 +1,27 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE GADTs #-}
+
+module DA.Ledger.Services.LedgerConfigurationService(getLedgerConfiguration) where
+
+import DA.Ledger.LedgerService
+import DA.Ledger.Stream
+import DA.Ledger.Types
+import Control.Concurrent (forkIO)
+import Network.GRPC.HighLevel.Generated
+import DA.Ledger.GrpcWrapUtils
+import DA.Ledger.Convert
+import qualified Com.Digitalasset.Ledger.Api.V1.LedgerConfigurationService as LL
+
+getLedgerConfiguration :: LedgerId -> LedgerService (Stream LedgerConfiguration)
+getLedgerConfiguration lid =
+    makeLedgerService $ \timeout config -> do
+    stream <- newStream
+    let request = LL.GetLedgerConfigurationRequest (unLedgerId lid) noTrace
+    _ <- forkIO $
```

Comment:
Might be a good idea to open a ticket for improving thread handling here so we don’t forget about it.

Reply:
I will deal with the thread handling in the next PR